templates/travel/partials/search-navbar/search-navbar.snip.html

```diff
@@ -0,0 +1,17 @@
+<section class="relative z2">
+  <header class="absolute top-0 right-0 left-0">
+    <!-- XS Breakpoint Navbar -->
+    <div class="sm-hide md-hide lg-hide px1 py1 flex justify-center items-center">
+      <a href="travel.amp" class="inline-block circle">
+        {{> ../icon/logo.snip.html}}
+      </a>
+    </div>
+
+    <!-- MD Breakpoint Navbar -->
```

Comment:
ok since this is a small piece of code , i was able to read and understand what you were doing and why. 

Here is what i would do in the scenario

```html
+    <div class="px1 md-px2 py1 md-py2 flex justify-between items-center">
 +      <a href="travel.amp" class="inline-block circle">
 +        {{> ../icon/logo.snip.html}}
 +      </a>
 +    </div>
```

USE CSS for XS- specific stuff, 

Once you start using duplicated DOM a lot, making even small edits to this will become impossible 

Duplicate DOM when you need different actions on different screen widths
(There might be some special case that warrants this but this should not be your baseline approach)

Reply:
Makes sense. I'll give this and future PRs a pass with this in mind.